lib/carto/visualization_exporter.rb

```diff
@@ -16,8 +16,7 @@ def export_table(user_table, folder, format, is_remote)
       url = sql_api_query_url(query, table_name, user_table.user, privacy(user_table), format)
       exported_file = "#{folder}/#{table_name}.#{format}"
 
-      @http_client.get_file(url, exported_file, ssl_verifypeer: false, ssl_verifyhost: 0)
-
+      file = @http_client.get_file(url, exported_file, ssl_verifypeer: false, ssl_verifyhost: 0)
```

Comment:
It might not be the best approach to override this file with another afterwards. Who cleans up the original `file`? It's a general rule we try to follow to never overwrite variables or parameters.

Could you name it differently?

Follow up question: The original code `exported_file` will be the downloaded file. How has that changed?